solana/src/main/java/com/solana/networking/RpcResponse.kt

```diff
@@ -16,10 +16,29 @@ typealias DefaultRpcResponse = RpcResponseSerializable<JsonElement>
 data class RpcError(val code: Int, val message: String)
 
 @Serializable
-open class RpcResponseSerializable<R>(
-    open val result: R? = null,
-    open val error: RpcError? = null,
-    val id: String? = null
+data class RpcResponseSerializable<R>(
```

Comment:
can this be renamed now?

Reply:
lets rename it

Reply:
It will affect your code on Metaplex but is better now.